### PR TITLE
CB-13420 If the CIDR duplicated on the provider then the  env list br…

### PR DESCRIPTION
**Pull request by topolyai5**
…oken

See detailed description in the commit message.

### CODE REVIEW

File: cloud-aws-cloudformation/src/test/java/com/sequenceiq/cloudbreak/cloud/aws/AwsNetworkConnectorTest.java [hunk 1]
```diff
@@ -283,6 +283,24 @@ public void testGetNetworkCidr() {
...
 
+    @Test
+    public void testGetNetworkCidrWithDuplicatedCidr() {
+        String existingVpc = "vpc-1";
+        String cidrBlock = "10.0.0.0/16";
+
+        Network network = new Network(null, Map.of(NetworkConstants.VPC_ID, existingVpc, "region", "us-west-2"));
+        CloudCredential credential = new CloudCredential();
+        AmazonEc2Client amazonEC2Client = mock(AmazonEc2Client.class);
+        DescribeVpcsResult describeVpcsResult = describeVpcsResult(cidrBlock);
```
Comment (bergerdenes) [inline cloud-aws-cloudformation/src/test/java/com/sequenceiq/cloudbreak/cloud/aws/AwsNetworkConnectorTest.java:294]: it does not test for duplicated cidrblock

**Review by bergerdenes — CHANGES_REQUESTED** [review]

**Review by bergerdenes — APPROVED** [review]

**Review by biharitomi — APPROVED** [review]
LGTM

**Review by doktoric — APPROVED** [review]

File: cloud-aws-cloudformation/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsNetworkConnector.java [hunk 2]
```diff
@@ -136,6 +136,7 @@ public NetworkCidr getNetworkCidr(Network network, CloudCredential credential) {
                 List<String> cidrs = vpc.getCidrBlockAssociationSet()
                         .stream()
                         .map(VpcCidrBlockAssociation::getCidrBlock)
+                        .distinct()
                         .collect(Collectors.toList());
                 LOGGER.info("The VPC {} CIDRs block are {}.", vpc.getVpcId(), cidrs);
                 vpcCidrs.addAll(cidrs);
```
Comment (doktoric) [inline cloud-aws-cloudformation/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsNetworkConnector.java:142]: do we need a distinct on the whole list?

Comment (topolyai5) [reply cloud-aws-cloudformation/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsNetworkConnector.java:142]: Where should we need distinct? I cannot distinct an element

Comment (topolyai5) [reply cloud-aws-cloudformation/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsNetworkConnector.java:142]: Ok, As we talked, I will distinct the vpcCidrs. Good catch

**Pull request merged**